appinventor/appengine/src/com/google/appinventor/client/wizards/youngandroid/RemixedYoungAndroidProjectWizard.java

```diff
@@ -77,7 +77,7 @@ public void execute() {
         // loading indicator will be hided or forced to be hided in gallery.loadSourceFile
         content.add(loading);
         popup.center();
-        boolean success = gallery.loadSourceFile(app, projectNameTextBox.getText(), popup);
+        boolean success = gallery.loadSourceFile(app, projectNameTextBox.getText().trim(), popup);
```

Comment:
Ditto.

Reply:
In this case, you could make use of the `projectName` variable since the value has already been computed.